### DISCUSSION TITLE
very large docker subnet network

### DISCUSSION THREAD

**Discussion by marcelobalbinot** [akvorado/akvorado#1502] [Q&A]
how to change Akvorado subnet to 172.18.0.1/24 instead of 172.18.0.1/16 ?

**Comment by vincentbernat** [comment]
Not really related to Akvorado, but there is a mention of that in the troubleshooting section of the documentation.

**Comment by marcelobalbinot** [comment]
solved, thanks for the help

root@akvorado:~$ cat /etc/docker/daemon.json
{
 "default-address-pools": [{"base":"240.0.0.0/16","size":24}],
 "userland-proxy": false
}


/etc/init.d/docker  restart

cd /opt/akvorado
docker compose down
docker compose up -d
